_template/chain.json

```diff
@@ -0,0 +1,105 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "",
+  "status": "live",
+  "website": "",
+  "network_type": "mainnet",
+  "pretty_name": "",
+  "chain_id": "",
+  "bech32_prefix": "",
+  "daemon_name": "",
+  "node_home": "$HOME/",
+  "key_algos": [
+    "secp256k1"
+  ],
+  "slip44": 118,
+  "fees": {
+    "fee_tokens": [
+      {
+        "denom": "",
+        "fixed_min_gas_price": 0,
+        "low_gas_price": 0,
+        "average_gas_price": 0.025,
+        "high_gas_price": 0.04
+      }
+    ]
+  },
+  "staking": {
+    "staking_tokens": [
+      {
+        "denom": ""
+      }
+    ]
+  },
+  "codebase": {
+    "git_repo": "",
+    "recommended_version": "v1.0",
+    "compatible_versions": [
+      "v1"
+    ],
+    "cosmos_sdk_version": "",
+    "cosmwasm_enabled": false,
+    "cosmwasm_path": "$HOME/.example/data/wasm",
+    "binaries": {
+      "linux/amd64": ""
+    },
+    "genesis": {
+      "genesis_url": ""
+    },
+    "versions": [
+      {
+        "name": "v1",
+        "recommended_version": "v1",
+        "compatible_versions": [
+          "v1"
+        ],
+        "cosmos_sdk_version": "",
+        "cosmwasm_enabled": false,
+        "binaries": {
+          "linux/amd64": ""
+        }
+      }
+    ]
+  },
+  "logo_URIs": {
+    "png": "",
```

Comment:
can we put what would be the full github url here, but for chain_name we can use _template, and for the image name, we can use chain_image.png. I think it'd help teams realize they need to upload the image and not refer to some remote location.